pkg/apis/crd/v1alpha1/types.go

```diff
@@ -403,12 +403,17 @@ type Rule struct {
 	// +optional
 	To []NetworkPolicyPeer `json:"to,omitempty"`
 	// Rule is matched if traffic is intended for a Service listed in this field.
-	// Currently only ClusterIP types Services are supported in this field. This field
+	// Currently, only ClusterIP types Services are supported in this field. This field
 	// can only be used when AntreaProxy is enabled. This field can't be used with To
 	// or Ports. If this field and To are both empty or missing, this rule matches all
 	// destinations.
 	// +optional
 	ToServices []NamespacedName `json:"toServices,omitempty"`
+	// Rule is matched if traffic is intended for a multi-cluster Service listed in
+	// this field. Service name and Namespace provided should match the original
+	// exported Service. This field can only be used when AntreaProxy is enabled.
+	// +optional
+	ToMulticlusterServices []NamespacedName `json:"toMulticlusterServices,omitempty"`
```

Comment:
I wonder if it looks neat to continue adding individual fields to the rule struct.
I was thinking about moving "ToServices" to "To" and make service one kind of NetworkPolicyPeer, like:
```
to:
- service:
    namespace: foo
    name: bar
```
I don't remember why we don't go this way in the beginning, but I don't think of big problem of this style.

If we do it for `toServices`, then we could leverage the Scope field in NetworkPolicyPeer, and it could be:
```
to:
- service:
    namespace: foo
    name: bar
  scope: ClusterSet
```

Does it make sense to you? @Dyanngg @jianjuns @antoninbas

Reply:
The problem is Ports and Protocols do not apply to Services?

But we can consider changing ToMulticlusterServices to what you described (a scope field of ToServices item). @luolanzone ?

Reply:
Added a new commit to do just that. I prefer to discuss if we need to move toServices in the next release cycle.

Reply:
I feel the way to change it as `ToServices` with a scope field looks neat.  @Dyanngg don't forget to update the PR summary and commit title and message.